tests/test_integrity.py

```diff
@@ -557,3 +571,91 @@ def test_remove_database(db):
     db.engine.dispose()
```

Comment:
The `test_remove_database` function deletes the database. There should be no tests after this.